frontend/src/metabase/reference/metrics/MetricDetail.jsx

```diff
@@ -56,208 +56,179 @@ const mapDispatchToProps = {
+                    description={entity.points_of_interest}
+                    placeholder={t`Nothing interesting yet`}
+                    isEditing={isEditing}
+                  />
+                </li>
+                <li className="relative">
+                  <Detail
+                    field={getField("caveats")}
+                    name={t`Things to be aware of about this metric`}
+                    description={entity.caveats}
+                    placeholder={t`Nothing to be aware of yet`}
+                    isEditing={isEditing}
+                  />
+                </li>
+                <li className="relative">
+                  <Detail
+                    field={getField("how_is_this_calculated")}
+                    name={t`How this metric is calculated`}
+                    description={entity.how_is_this_calculated}
+                    placeholder={t`Nothing on how it's calculated yet`}
+                    isEditing={isEditing}
+                  />
+                </li>
+                {table && !isEditing && (
                   <li className="relative">
-                    <Detail
-                      id="points_of_interest"
-                      name={t`Why this metric is interesting`}
-                      description={entity.points_of_interest}
-                      placeholder={t`Nothing interesting yet`}
-                      isEditing={isEditing}
-                      field={points_of_interest}
+                    <Formula
+                      type="metric"
+                      entity={entity}
+                      isExpanded={isFormulaExpanded}
+                      expandFormula={expandFormula}
+                      collapseFormula={collapseFormula}
                     />
                   </li>
-                  <li className="relative">
-                    <Detail
-                      id="caveats"
-                      name={t`Things to be aware of about this metric`}
-                      description={entity.caveats}
-                      placeholder={t`Nothing to be aware of yet`}
-                      isEditing={isEditing}
-                      field={caveats}
-                    />
-                  </li>
-                  <li className="relative">
-                    <Detail
-                      id="how_is_this_calculated"
-                      name={t`How this metric is calculated`}
-                      description={entity.how_is_this_calculated}
-                      placeholder={t`Nothing on how it's calculated yet`}
-                      isEditing={isEditing}
-                      field={how_is_this_calculated}
+                )}
+                {!isEditing && (
+                  <li className="relative mt4">
+                    <FieldsToGroupBy
+                      fields={table.fields
+                        .map(fieldId => metadataFields[fieldId])
+                        .reduce(
```

Comment:
not related to this PR, but I discovered that this sometimes gives you fields that you can't actually group by: like JSON fields.